### PR TITLE
[qa] Add tests verifychain, lockunspent, getbalance, listsinceblock

### PR DESCRIPTION
**Pull request by maflcko**

### CODE REVIEW

File: qa/rpc-tests/mempool_limit.py [hunk 1]
```diff
@@ -38,7 +38,6 @@ def run_test(self):
         self.nodes[0].settxfee(0) # return to automatic fee selection
         txFS = self.nodes[0].signrawtransaction(txF['hex'])
         txid = self.nodes[0].sendrawtransaction(txFS['hex'])
-        self.nodes[0].lockunspent(True, [us0])
```
Comment (MarcoFalke) [inline qa/rpc-tests/mempool_limit.py]: @jonasschnelli I fail to see why you added this noop. I guess you used it for debugging and it is save to remove.

Comment (jonasschnelli) [inline qa/rpc-tests/mempool_limit.py]: Yes. I guess this can be removed. Should be locked by L39 anyways.

Comment (MarcoFalke) [inline qa/rpc-tests/mempool_limit.py]: `lockunspent(True)` will *un*lock, but signrawtransaction does not add any lock.

**Comment by laanwj** [comment]
ACK fa4a522

**Pull request merged**